fv3core/utils/grid.py

```diff
@@ -1141,28 +1141,28 @@ def _old_grid_axis_offsets(
         origin_offset = grid.is_ - origin[0]
         i_start = gtscript.I[0] + proc_offset + origin_offset
     else:
-        i_start = gtscript.I[0] - np.iinfo(np.int32).max
+        i_start = gtscript.I[0] - np.iinfo(np.int16).max
```

Comment:
Poke @mcgibbon  (changes that will touch upon the Grid init work)

Reply:
This is in `grid.py` but it shouldn't affect the grid init, this is mainly for regions to be excluded.

I'm curious for my knowledge though about why the change from int32 to int16?

Reply:
We are using `int32` in the GTC implementation, so using `int16` in `fv3core` ensures these numbers will be smaller in magnitude than the toolchain endpoints.

Reply:
Could you add a comment to that effect? It's impossible to tell otherwise, which might lead to someone reverting it.